appveyor.yml

```diff
@@ -0,0 +1,42 @@
+version: 1.0.{build}
+max_jobs: 2
+image: Visual Studio 2015
+build_script:
+- cmd: >-
+    git submodule init
+
+    git submodule update
+
+    nuget restore ReactWindows\ReactNative.sln
+
+    set DEVENV="%VS140COMNTOOLS%\..\IDE\devenv"
+
+    %DEVENV% /build "Debug|x64" ReactWindows\ReactNative.sln
+
+    npm install -g npm
+
+    npm version
+
+    npm install -g react-native-cli
+
+    npm install
+
+    mkdir ReactWindows\Playground\ReactAssets
+
+    react-native bundle --platform windows --entry-file ReactWindows/Playground/index.windows.js --bundle-output ReactWindows/Playground/ReactAssets/index.windows.bundle --assets-dest ReactWindows/Playground/ReactAssets/
```

Comment:
I had it in my build for two reasons:
* To make sure upstream republishes with same version didn't become incompatible with the Playground index.*.js
* To have Playground ReleaseBundle artefacts at the end of the build that were ready to run for basic UI tests that would verify the "Welcome to React Native" message (versus a redbox, which would be a regression).

Happy to ditch the step for the main repo build.